### ISSUE TITLE
No difference even when delay is at max of 957ms

### ISSUE THREAD

**Issue by UNIVAC-Colonel-Panic** [CemraJC/clickfix#13]
ClickFix is Enabled
Windows 10 Home
[Logitech M720 Mouse using Unifying Receiver](https://support.logitech.com/en_us/product/m720-triathlon)

How do I troubleshoot this?

**Comment by CemraJC** [comment]
Hi! Sorry for the terrible response delay - I imagine you would have replaced your mouse by now 😬 

If anyone else has this issue, head to www.clickfix.cf and scroll to the bottom of the page, there should be a tester there to show you what events are being detected. If you're seeing no change when using ClickFix, it may be that your system has an extra tight security configuration or policy that doesn't allow the ClickFix program to hook into your mouse events.

Let me know if you're still having issues, post your results from the click tester before/after and hopefully there's something to go off!

**Comment by vvinjj** [comment]
Hi @CemraJC, thanks for your repository. I had same issue with my Mad Catz R.A.T. 7 mouse on windows 10. Had to modify the script logic to make it working in my case:
`#SingleInstance, force
#InstallMouseHook
SendMode, Input
threshold := 100
lastD := 0
lastU := 0
isUp := True
isProcessing := False

LButton::
Critical
if (isProcessing) {
    return
} Else {
    isProcessing = True
}
if (isUp && A_TickCount - lastU > threshold && A_TickCount - lastD > threshold) {    
    lastD := A_TickCount
    isUp := False
    Send {Blind}{LButton Down}
}
isProcessing := False
return

LButton up::
if (isProcessing) {
    return
} Else {
    isProcessing = True
}
if (!isUp && A_TickCount - lastU > threshold) {
    lastU := A_TickCount
    isUp := True
    Send {Blind}{LButton Up}
}
isProcessing := False
return
`
Basically, filter out parallel event calls invocations and up vs down logical order filter. Note, for some reason ClickFix-Logger.ahk no longer catching the events (had no time to investigate), but mouse behaves now as it should.

**Comment by ilaell** [comment]
I got the same Problem, max setting and still doubleklicks
but i dont know anything about this "Magic" :) so i need a bit help
